fastlane/Fastfile

```diff
@@ -397,6 +397,8 @@ lane :create_github_release do |options|
   sh "gem push --key github --host https://rubygems.pkg.github.com/#{github_username} #{gem_path}" unless options[:skip_github_packages]
 
   sh "gem push #{gem_path}" unless options[:skip_rubygems]
+
+  update_docs(version: version) unless options[:skip_docs]
```

Comment:
Re-running to fix docs fail because the gem already exists. There is a check in beginning of function that early returns if Gem exists. We need to change gating or make a part 3 solely for docs